src/common/reliability/Image-Segmentation/maskrcnn_resizeimage.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# coding: utf-8
+import sys
+import numpy as np
+from imgseg.predict import predict, show
+from torchvision.models.detection import maskrcnn_resnet50_fpn
+from PIL import Image
+import time
+# from IPython.display import display
+# from utils.ipstartup import *
+sys.path.insert(0, "utils")
+
+model = maskrcnn_resnet50_fpn(pretrained=True)
+
+# Applies MaskRCNN on downscaled image, by default the factor is 10x
+def predictByResize(image, flag=0, factor=10):
```

Comment:
Use docstrings to document functions
```suggestion
def predictByResize(image, flag=0, factor=10):
    """Applies MaskRCNN on downscaled image, by default the factor is 10x"""
```